openpaas-james/apps/distributed/src/main/java/com/linagora/openpaas/james/app/BlobStoreCacheModulesChooser.java

```diff
@@ -0,0 +1,83 @@
+package com.linagora.openpaas.james.app;
```

Comment:
Isn't that file copied from apache project? You need to keep the apache license on top then

Idem for similar files